albatross/covariance_functions/distance_metrics.h

```diff
@@ -52,6 +52,18 @@ class RadialDistance : public DistanceMetric {
   }
 };
 
+class AngularDistance : public DistanceMetric {
+public:
+  AngularDistance(){};
```

Comment:
nit: Do we need to define the default constructor?

Reply:
Nice catch, right we don't need it since it's inherited from `DistanceMetric`.